contrib/babelfishpg_tds/src/backend/tds/tdstypeio.c

```diff
@@ -1963,6 +1979,49 @@ TdsRecvTypeDatetime2(const char *message, const ParameterToken token)
 	return result;
 }
 
+/* -------------------------------
+ * TdsRecvTypeGeometry - converts external binary format to
+ * Geometry data type
+ * --------------------------------
+ */
+Datum
+TdsRecvTypeGeometry(const char *message, const ParameterToken token)
+{
+	Datum result = 0; 
+
+	//Decode binary and convert if needed 
+	StringInfo	buf = TdsGetStringInfoBufferFromToken(message, token);
```

Comment:
Why even bother for this if we are going to throw an error eventually?

Reply:
It is currently a placeholder function, So just copied the template,
although added a comment for placeholder function, will addressing this in subsequent PR's